### PR TITLE
write rspec view: admin/courses/index.html.erb

### PR DESCRIPTION
**Pull request by duypn-2245**

### CODE REVIEW

**Comment by duypn-2245** [comment]
image

**Comment by duypn-2245** [comment]
image

**Comment by duypn-2245** [comment]
Redmine ticket link: https://edu-redmine.sun-asterisk.vn/issues/26346

**Comment by duypn-2245** [comment]
ready

File: app/views/admin/courses/index.html.erb [hunk 1]
```diff
@@ -1,7 +1,10 @@
 <%= provide :title, t("admin.courses.visit.title") %>
-<h3 class=""><%= t("course.visit.title") %></h3>
-<%= will_paginate %>
+<h3><%= I18n.t "admin.courses.visit.title" %></h3>
 <ul class="courses">
-  <%= render @courses %>
+  <% if  @courses %>
```
Comment (vanvtt-0952) [inline app/views/admin/courses/index.html.erb]: rảnh vãi chú chỉ cần ` <%= render @courses || t(".no_avaiable") %>`

**Comment by duypn-2245** [comment]
ready

**Comment by vanvtt-0952** [comment]
merged

**Pull request merged**